UI/RELEASE-NOTES

```diff
@@ -1,3 +1,6 @@
+*Version 4.5.3*
+* Upgrade ejs module from 3.1.6 to 3.1.9 because of listed vulnerability
```

Comment:
@ManuelCalvo3Pillar Can you change library name and versions on release notes to workbox-webpack-plugin

Reply:
done